nautobot/docs/administration/nautobot-server.md

```diff
@@ -338,6 +338,12 @@ Removing expired sessions...
 Invalidating cache...
 ```
 
+### `remove_stale_scheduled_jobs`
+
+`nautobot-server remove_stale_scheduled_jobs [max-age of days]`
+
+Delete non-recurring scheduled jobs that were scheduled to run more than `max-age` days ago.
+
```

Comment:
Would you mind drafting an addition to the release-notes explaining the `has_sensitive_variables` and `remove_stale_scheduled_jobs` features?